tests/test_permissions_synchronizer.py

```diff
@@ -1,125 +1,98 @@
-from cowbird.services.service_factory import ServiceFactory
+from cowbird.config import MULTI_TOKEN, SINGLE_TOKEN, ConfigErrorInvalidResourceKey, ConfigErrorInvalidTokens
+from cowbird.services.impl.magpie import MAGPIE_ADMIN_PASSWORD_TAG, MAGPIE_ADMIN_USER_TAG
 from tests import utils
 
 
+def check_config_raises(config_data, exception_type):
+    cfg_file = tempfile.NamedTemporaryFile(mode="w", suffix=".cfg", delete=False)
+    with cfg_file as f:
+        f.write(yaml.safe_dump(config_data))
+    utils.check_raises(lambda: utils.get_test_app(settings={"cowbird.config_path": f.name}),
+                       exception_type, msg="invalid config file should raise")
+    os.unlink(cfg_file.name)
+
+
 @pytest.mark.permissions
-class TestSyncPermissions(unittest.TestCase):
+class TestSyncPermissionsConfig(unittest.TestCase):
     """
-    Test permissions synchronization.
+    Test config for permissions synchronization
     """
-
-    @classmethod
-    def setUpClass(cls):
-        service1 = "Geoserver"
-        service2 = "Thredds"
-        cls.test_services = [service1, service2]
-        cls.res_root = {service1: "/api/workspaces/private/",
-                        service2: "/catalog/birdhouse/workspaces/private/"}
-        cls.sync_perm_name = {service1: ["read"],
-                              service2: ["read", "browse"]}
-        mapping_point_1 = "mapping_point_1"
-        cls.mapped_service = {service1: service2,
-                              service2: service1}
-        data = {
+    def setUp(self):
+        self.data = {
             "services": {
-                "Magpie": {"active": True, "url": ""},
-                service1: {"active": True},
-                service2: {"active": True}
-            },
-            "sync_permissions": {
-                mapping_point_1: {
-                    "services": {
-                        service1: cls.res_root[service1],
-                        service2: cls.res_root[service2]
-                    },
-                    "permissions_mapping": [
-                        {
-                            service1: cls.sync_perm_name[service1],
-                            service2: cls.sync_perm_name[service2]
-                        }
-                    ]
-                }
+                "Magpie": {"active": True, "url": "",
+                           MAGPIE_ADMIN_USER_TAG: "admin", MAGPIE_ADMIN_PASSWORD_TAG: "qwertyqwerty"},
+                "Thredds": {"active": True}
             }
-
         }
-        cls.cfg_file = tempfile.NamedTemporaryFile(mode="w", suffix=".cfg", delete=False)
-        with cls.cfg_file as f:
-            f.write(yaml.safe_dump(data))
-        cls.app = utils.get_test_app(settings={"cowbird.config_path": cls.cfg_file.name})
-
-    @classmethod
-    def tearDownClass(cls):
-        utils.clear_services_instances()
-        os.unlink(cls.cfg_file.name)
-
-    def test_sync(self):
```

Comment:
Previous test seemed to verify the sync operation itself, not just the config that defined sync conditions.

Why are those checks removed?